include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -149,7 +149,7 @@ namespace picongpu
                      * The openPMD plugin only supports configuring writing routines via JSON.
                      * Reading routines get an empty JSON set.
```

Comment:
This comment must be updated to reflect the changes in this PR.

Reply:
done